acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
thanks.  i forgot.  these changes were required when making the helper run the apply,module,config tests on the master.

i'll add agent roles back on the masters locally and see if the changes to the helper fixes your issues.